src/transformers/searchservice/ResultsFactory.ts

```diff
@@ -137,4 +139,17 @@ export class ResultsFactory {
       entityType: result.type,
     };
   }
+
+  public static fromFunctionVertical(result: any, index: number): Result {
```

Comment:
Can we put this next to "fromCustomSource"? Plus we probably could refactor the two methods to use a helper since the only difference if the source between the two of them :)